content/chainguard/administration/custom-idps/okta/index.md

```diff
@@ -29,11 +29,13 @@ To complete this guide, you will need the following.
 
 To integrate your Okta identity provider with the Chainguard platform, [log in to Okta](https://www.okta.com/login/) and navigate to the **Applications** landing page in the Admin console. There, click the **Create App Integration** button.
 
-![Screenshot of the Okta Admin console, showing the Applications landing page. "Applications" is circled in the left hand sidebar menu, as is the "Create App Integration" button.](okta-1.png)
+<center><img src="okta-1.png" alt="Screenshot of the Okta Admin console, showing the Applications landing page. 'Applications' is circled in the left hand sidebar menu, as is the 'Create App Integration' button." style="width:950px;"></center>
+<br /> 
 
 Select **OIDC - OpenID Connect** as the sign-in method and **Web Application** as the application type.
 
-![Screenshot of the Okta Admin console, showing the "Create a new app integration" modal window. This image shows the "OIDC - Open ID Connect" option selected next to "Sign-in Method", and the "web Application" option selected next to "Application type."](okta_2_create_buttons.png)
+<center><img src="okta-2.png" alt="Screenshot of the Okta Admin console, showing the 'Create a new app integration' modal window. This image shows the 'OIDC - Open ID Connect' option selected next to 'Sign-in Method', and the 'web Application' option selected next to 'Application type'." style="width:950px;"></center>
```

Comment:
This image is giving a 404. From the okta folder, would it be `okta_2_create_buttons.png` ?

Reply:
I just renamed all the images in that folder so they're not so funky, thanks for catching!